payu/models/cice.py

```diff
@@ -201,9 +194,55 @@ def setup(self):
             setup_nml['runtype'] = 'continue'
             setup_nml['restart'] = True
 
+        else:
+            # Locate and link any restart files (if required)
+            if not setup_nml['ice_ic'] in ('none', 'default'):
+                self.link_restart(setup_nml['ice_ic'])
+
+            if setup_nml['restart']:
+                self.link_restart(setup_nml['pointer_file'])
+
+        self.calc_runtime(setup_nml)
+
+        # Write any changes to the work directory copy of the cice
+        # namelist
+        nml_path = os.path.join(self.work_path, self.ice_nml_fname)
+        self.ice_in.write(nml_path, force=True)
+
+    def calc_runtime(self, setup_nml):
+        """
+        Calculate 1: the previous number of timesteps simulated, and 2:
+        the number of timesteps to simulate in the next run.
+
+        Note 1: This method is overridden in the cice5 driver, as cice5
+        in ACCESS OM2 does not require the calculated runtime information.
+        Instead it performs the calculations within the model using the
+        restart files and OM2 namelist.
```

Comment:
```suggestion
        Note 1: This method is overridden in the cice5 driver.
```

I think this is enoguh detail?

Reply:
Yeah I think so! I'll commit this along with the other suggestions.